ospfd/ospf_opaque.c

```diff
@@ -1598,6 +1604,9 @@ ospf_opaque_lsa_install (struct ospf_lsa *lsa, int rt_recalc)
         }
       break;
     case OSPF_OPAQUE_AS_LSA:
+      if (IS_DEBUG_OSPF_TRACE)
```

Comment:
Isn't this IS_DEBUG_OSPF (lsa, LSA_INSTALL)?  Why does this case statement get a trace but the others dont?